engine/apps/slack/models/slack_message.py

```diff
@@ -117,7 +126,9 @@ def permalink(self) -> typing.Optional[str]:
 
     @property
     def deep_link(self) -> str:
-        return f"https://slack.com/app_redirect?channel={self.channel.slack_id}&team={self.slack_team_identity.slack_id}&message={self.slack_id}"
+        # TODO: once _channel_id has been fully migrated to channel, remove _channel_id
+        # see https://raintank-corp.slack.com/archives/C06K1MQ07GS/p173255546
+        return f"https://slack.com/app_redirect?channel={self._channel_id}&team={self.slack_team_identity.slack_id}&message={self.slack_id}"
```

Comment:
"fix" from https://github.com/grafana/oncall/pull/5292 (let's wait to read from `channel` until the data-migration mentioned in that PR has been fully finished)